packages/ckeditor5-dev-utils/lib/translations/multiplelanguagetranslationservice.js

```diff
@@ -368,3 +388,28 @@ module.exports = class MultipleLanguageTranslationService extends EventEmitter {
 		return path.join( pathToPackage, 'lang', 'translations' );
 	}
 };
+
+/**
+ *
+ * @param {String|((name: string) => boolean)|RegExp} predicate
+ * @param {String[]} options
+ * @returns {String|undefined}
+ */
```

Comment:
`String|RegExp|Function` would be enough. OTOH, why do we need to support such complex construction?

Reply:
`s/String[]/Array.<String>`

Reply:
We need to support at least strings and regexps. We need strings for specifying the name that might not exist and regexp for matching the existing name. I added support for function as it always allows for full customization.

Reply:
I would put the comment in the code. In the future, it might help when we will be wondering why it's done this way and what we wanted to resolve.